prowler/providers/kubernetes/lib/service/service.py

```diff
@@ -0,0 +1,18 @@
+import threading
+
+from prowler.providers.kubernetes.kubernetes_provider_new import KubernetesProvider
+
+
+class KubernetesService:
+    def __init__(self, provider: KubernetesProvider):
+        self.context = provider.context
+        self.api_client = provider.api_client
+
+    def __threading_call__(self, call, iterator):
+        threads = []
+        for value in iterator:
+            threads.append(threading.Thread(target=call, args=(value,)))
+        for t in threads:
+            t.start()
+        for t in threads:
+            t.join()
```

Comment:
Please use the same threading logic as AWS -> https://github.com/prowler-cloud/prowler/blob/0fff0568fab3b79202de71ff277aa0f9fb18e809/prowler/providers/aws/lib/service/service.py#L55

Reply:
Done